internal/resource/resource.go

```diff
@@ -60,6 +60,10 @@ func AnyResourceFromConfigurationV1(c *model.Configuration) model.AnyResource {
 		a.Spec["destinations"] = c.Spec.Destinations
 	}
 
+	if len(c.Spec.Extensions) > 0 {
+		a.Spec["extensions"] = c.Spec.Extensions
+	}
+
```

Comment:
This adds the extensions to the configuration spec Extensions field. The returned model.AnyResource is what will be send to BindPlane's `/apply` endpoint.